<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_room</class>
 <widget class="QDialog" name="main_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #d9e7ff, stop:1 #e3ffe7);
/*background: linear-gradient(90deg, #e3ffe7 0%, #d9e7ff 100%);</string>
  </property>
  <widget class="QPushButton" name="logout_main_room_Btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 50px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>â†©</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_pw_Btn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>0</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 15px;
	font-weight: 600;
	/*background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);*/
	/*background: linear-gradient(90deg, #e3ffe7 0%, #d9e7ff 100%);*/
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
	/*background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
	/*background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
	/*background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 255, 255));*/
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="living_room_Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	qproperty-icon: url(:/Images/Living.png);
	qproperty-iconSize: 40px 40px;
	text-align: center;
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	font-weight: 700;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);

}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Living room</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bed_room_Btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	qproperty-icon: url(:/Images/Bed.png);
	qproperty-iconSize: 40px 40px;
	text-align: center;
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	font-weight: 700;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);

}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Bed room</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kitchen_room_Btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>265</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	qproperty-icon: url(:/Images/Kitchen.png);
	qproperty-iconSize: 40px 40px;
	text-align: center;
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	font-weight: 700;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);

}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Kitchen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bath_room_Btn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>265</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	qproperty-icon: url(:/Images/Bath.png);
	qproperty-iconSize: 40px 40px;
	text-align: center;
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	font-weight: 700;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);

}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Bath room</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_living">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>440</y>
     <width>250</width>
     <height>148</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);
border-style: solid;
border-width: 2px;
border-color: #9999aa;
color: rgb(255, 255, 255);
text-align: center;
font-size: 30px;
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Living</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_garage">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>440</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);
border-style: solid;
border-width: 2px;
border-color: #9999aa;
color: rgb(255, 255, 255);
text-align: center;
font-size: 30px;
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Garage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bed">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>440</y>
     <width>250</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 0);
border-style: solid;
border-width: 2px;
border-color: #9999aa;
color: rgb(255, 255, 255);
text-align: center;
font-size: 30px;
font-weight: 700;</string>
   </property>
   <property name="text">
    <string>Bed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera_Btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
border-image: url(:/Images/camera.png);</string>
   </property>
   <property name="text">
    <string>â†©</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_time">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>170</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 30px;
border-style: solid;
border-width: 2px;
border-color: #9999aa;
border-radius: 10px;
background-color: none;
text-align: center;
font-weight: 700;
color: rgb(0, 0, 0);

</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_temp">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Temp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_doam">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Doam</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_off_wifi_Btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	/*background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);*/
	background-color: rgb(85, 255, 0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wifi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Wifi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_key">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: none;
color: rgb(0, 0, 0);
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="on_off_key_Btn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-style: solid;
	border-width: 2px;
	border-color: #9999aa;
	border-radius: 10px;
	color: white;
	font-size: 20px;
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
}

QPushButton:enabled {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #000851, stop:1 #1CB5E0);
		color: white;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0  rgba(0, 255, 255, 255),, stop:1 #1CB5E0);
		color: #fffffe;
}


QPushButton:hover:!pressed {
	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:1, stop:0 #e3ffe7, stop:1 #d9e7ff);
		color: #0c2f70;
}

QPushButton:disabled {
		background-color: #aaaaaa;
		color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
